index.html

```diff
@@ -106,6 +110,7 @@ <h1 class="display-4 fw-bold lh-1 mb-3">Contact TripleMKInc</h1>
     </div>
 </div>
 
+
```

Comment:
here also.

Reply:
I am going to resolve it in the next Pull Request.